client/actions/planning/api.js

```diff
@@ -967,6 +967,24 @@ const markPlanningPostponed = (plan, reason) => ({
     },
 })
 
+/**
+ * Export selected planning items as a new article
+ */
+function exportAsArticle() {
+    return (dispatch, getState, { api, notify, gettext }) => {
+        const state = getState()
+
+        return api.save('planning_export', {
```

Comment:
can we display the error if the call fails?

Reply:
sure, will add it

Reply:
added just generic error message, not sure atm what could get wrong :wink: